src/Atmos/Model/remainder.jl

```diff
@@ -47,11 +47,47 @@ function flux_nondiffusive!(rem::RemainderModel, flux::Grad, state::Vars, aux::V
   flux_s = similar(flux)
   m_s = getfield(flux_s, :array)
 
-  for sub in rem.subs
-    fill!(m_s, 0)
-    flux_nondiffusive!(sub, flux_s, state, aux, t)
-    m .-= m_s
-  end
+  fill!(m_s, 0)
+  flux_nondiffusive!(rem.sub, flux_s, state, aux, t)
+  m .-= m_s
+  nothing
+end
+
+function flux_nondiffusive!(rem::RemainderModel{M,S}, flux::Grad, state::Vars, aux::Vars, t::Real) where {M,S<:AtmosAcousticLinearModel}
+  FT = eltype(state)
+  ρ = state.ρ
+  ρu = state.ρu
+  ρe = state.ρe
+  ref = aux.ref_state
+  u = ρu / ρ
+  e_pot = gravitational_potential(rem.main.orientation, aux)
+  p = pressure(rem.main.moisture, rem.main.orientation, state, aux)
+  # TODO: use MoistThermodynamics.linearized_air_pressure 
+  # need to avoid dividing then multiplying by ρ
+  pL = ρ * FT(R_d) * FT(T_0) + FT(R_d) / FT(cv_d) * (ρe - ρ * e_pot)
+
+  flux.ρ = -zero(FT)
+  flux.ρu = ρu .* u' + (p - pL) * I
+  flux.ρe = ((ρe + p) / ρ - (ref.ρe + ref.p) / ref.ρ + e_pot) * ρu
```

Comment:
I think this should be 
```suggestion
  flux.ρe = ((ρe - ref.ρe + p) / ρ + ref.p / ref.ρ + e_pot) * ρu
```